### DISCUSSION TITLE
PostgreSQL version 110016 is not supported

### DISCUSSION THREAD

**Discussion by gabitavanatorul** [Mimetis/Dotmim.Sync#999] [Q&A]
hi, I'm trying to syncronize a postgresql database (server) with a sqlite database (client), but while doing that im getting this error:

[NeedsToUpgradeAsync]..Engine PostgreSQL version 110016 is not supported. Please upgrade your server to the last version.
  -----------------------

  └  ─ Engine PostgreSQL version 110016 is not supported. Please upgrade your server to the last version.

Is there a way to make it work with this current postgresql version? Thanks!

**Comment by Mimetis** [comment]
No Sorry
